src/main/resources/mturk/worker-ui/mturk.js

```diff
@@ -30,11 +54,16 @@ function initMturk(platformName, workerServiceUrl, experiment) {
         experiment: experiment,
         preview: preview
     });
+
     WorkerUI.onSubmitAnswer(function (viewData, submittedData) {
-        //TODO log this
+        console.log("Save answer "+submittedData.answer);
+        creativeAnswers.push(submittedData.answer)
     });
-    WorkerUI.onSubmitCalibration(function (data) {
-        //TODO log this
+    WorkerUI.onSubmitRating(function (data) {
```

Comment:
should be `function (viewData, submittedData)`. I know it's the same in the sample, but i updated that ;)
viewData will return data used to display the view, submittedData will be what the worker-service gets posted